src/backends/backend.js

```diff
@@ -41,23 +41,28 @@ class LocalStorageAuthStore {
   }
 }
 
-const slugFormatter = (template = "{{slug}}", entryData, slugConfig) => {
-  const date = new Date();
-
-  const getIdentifier = (entryData) => {
-    const validIdentifierFields = ["title", "path"];
-    const identifiers = validIdentifierFields.map((field) =>
-      entryData.find((_, key) => key.toLowerCase().trim() === field)
-    );
+const validIdentifierFields = ["title", "path"];
 
-    const identifier = identifiers.find(ident => ident !== undefined);
+const getIdentifierKey = entryData => {
+  const keys = validIdentifierFields.map(field => {
+    return entryData.findKey((_, key) => {
+      return key.toLowerCase().trim() === field;
+    });
+  });
+  return keys.find(key => entryData.get(key) !== undefined);
```

Comment:
Is there a reason we can't use the old logic here, where we return the values (on line 48), instead of having to run `entryData.get` again here?

Reply:
```js
const keyVals = validIdentifierFields.map(field => {
  return entryData.find((_, key) => {
    return key.toLowerCase().trim() === field;
  });
});
return keyVals.find(key => key !== undefined);
```

Reply:
Agreed, I believe I was thinking we were lowercasing and trimming the return value for some reason, but that isn't the case.